cmd/helm/install.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package main
 
 import (
+	"bytes"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 
+	"html/template"
```

Comment:
I think we want `text/template`, though I'm not sure that it will make a difference in this case.